src/service/EmailSend.java

```diff
@@ -0,0 +1,63 @@
+package service;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+import javax.mail.Authenticator;
+import javax.mail.Message;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+public class EmailSend {
+
+	public void details(RegisterEnp registerEnp) throws IOException {
+		System.out.print("check");
+		String message = "First Name" + registerEnp.getFirstName() +
+				"\n  LastName" + registerEnp.getLastName() + "\n Password " + registerEnp.getPassword() + "\n Address "
+				+ registerEnp.getAddress() + "\n City " + registerEnp.getCity() + "\n Zip Code " + registerEnp.getZip()
+				+ "\n State " + registerEnp.getState() + "\n Country " + registerEnp.getCountry();
+		String to = registerEnp.getEmail();
+		String subject = "Confirmation";
+		Properties properties1 = new Properties();
+
+		FileInputStream fileInputStream = new FileInputStream("C:\\Users\\BHASKAR\\Desktop\\internship\\AdvProject2\\SaveFile2.Properties");
```

Comment:
Instead of putting file in the file system, you should put the property file in the project itself. Just remove your username and password before you commit it to the repository.